.bazelci/presubmit.yml

```diff
@@ -62,6 +62,18 @@ tasks:
     bazel: 4.1.0
     shell_commands:
       - "./test_coverage.sh"
+  test_bazel_52_linux:
+    name: "./test_bazel_5_2"
+    platform: ubuntu2004
+    bazel: 5.2.0rc2
+    shell_commands:
+      - "./test_bazel_5_2.sh"
+  test_bazel_52_linux:
```

Comment:
Is this duplicated intentionally?

Reply:
OOps, one of them need to be macos - thanks!